packages/paste-design-tokens/tokens/aliases/box-shadow.yml

```diff
@@ -4,6 +4,6 @@ imports:
   - ./spacing.yml
 aliases:
   shadow-elevation-0: "none"
-  shadow-elevation-10: "{!offset-0} {!offset-10}px {!offset-20}px {!offset-0} {!palette-gray-100-transparent-30}"
-  shadow-elevation-20: "{!offset-0} {!offset-40}px {!offset-60}px {!offset-0} {!palette-gray-100-transparent-20}"
-  shadow-elevation-30: "{!offset-0} {!offset-50}px {!offset-80}px {!offset-20} {!palette-gray-100-transparent-20}"
+  shadow-elevation-10: "{!offset-0} {!offset-10}px {!offset-40}px {!offset-0} {!palette-gray-100-transparent-10}"
+  shadow-elevation-20: "{!offset-0} {!offset-20}px {!offset-60}px {!offset-0} {!palette-gray-100-transparent-20}"
+  shadow-elevation-30: "{!offset-0} {!offset-60}px {!offset-80}px {!offset-20} {!palette-gray-100-transparent-20}"
```

Comment:
```suggestion
  shadow-elevation-30: "{!offset-0} {!offset-60}px {!offset-80}px {!offset-20}px {!palette-gray-100-transparent-20}"
```

Reply:
🤔 Should we just make the offset aliases strings that include the `px`?
![image](https://user-images.githubusercontent.com/1592327/89206777-5c0adc80-d57f-11ea-9126-52e4993b793c.png)
More consistent with the other tokens and removes the need to comment on the offset-0